src/librbd/cache/pwl/ssd/WriteLog.cc

```diff
@@ -765,16 +754,12 @@ void WriteLog<I>::append_ops(GenericLogOperations &ops, Context *ctx,
     span_payload_len += log_entry->write_bytes();
   }
   if (!span_payload_len || !log_entries.empty()) {
-    if (log_entries.size() > 1) {
-      bytes_to_free += (log_entries.size() - 1) * MIN_WRITE_ALLOC_SSD_SIZE;
-    }
-    write_log_entries(log_entries, aio, new_first_free_entry);
+      write_log_entries(log_entries, aio, new_first_free_entry);
   }
 
   {
     std::lock_guard locker1(m_lock);
     m_first_free_entry = *new_first_free_entry;
-    m_bytes_allocated -= bytes_to_free;
```

Comment:
@MahatiC should chime in but I think removing this is wrong.  Here `m_bytes_allocated` is adjusted after it is incremented according to a worse case estimate where each log entry needs its own control block and takes an extra `MIN_WRITE_ALLOC_SSD_SIZE` bytes of space:
https://github.com/ceph/ceph/blob/4e6fbccea448124a868af72bf16f115798d4b8d1/src/librbd/cache/pwl/ssd/WriteLog.cc#L322-L325

Reply:
@idryomov . Yes you are right.  I'll drop this patch. Thanks.